src/System.Management.Automation/engine/Subsystem/FeedbackSubsystem/IFeedbackProvider.cs

```diff
@@ -287,13 +288,14 @@ internal GeneralCommandErrorFeedback()
                     .AddParameter("ExpandProperty", "Name")
                 .Invoke<string>();
 
-            if (results.Count > 0)
-            {
-                return new FeedbackItem(
-                    SuggestionStrings.Suggestion_CommandNotFound,
-                    new List<string>(results),
-                    FeedbackDisplayLayout.Landscape);
-            }
+			if (results.Count > 0)
+			{
+				ApplicationInsightsTelemetry.SendUseTelemetry("PSCommandNotFoundSuggestion", "CommandNotFound");
```

Comment:
Now that there are multiple "command not found" feedback providers, it's better to use a more concrete name for this one, such as
```suggestion
				ApplicationInsightsTelemetry.SendUseTelemetry("FuzzyMatching", "CommandNotFound");
```